domains/asteral.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Asteral1",
+           "email": "",
+           "discord": "870366927653056582",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.F2v6VwlZy8PN3eAm0sdM9Kqvh7Cah2AqvxrUMBqNmUI0p-bZbD4Ovfjm2Ti_5SOJB2CF_Ff-2Fb1oaJfef7OeUL9YIdu2O2jFqUR81znaNuPoT9VpXEIAeIQwBvQylSRqnhZK92QALGNUOIo8_9scBEZ-4K1Sfgn6ypISO-O5en13fcYW5gjX_CdAy1CjzHU7PZ0EuF-Ym0G5FXj6gGY3AbysoSDAef38r7uzFbUKEbcRd6QUjMoQkGp9ST9NqP_-Mg0XFEPC9Bbyc6lvGwe2DbIoNzUqlo3b6nViKSiS0UIXe_cT0aImPRqvR4N5wSvwR3XK1R2mY0EewgsTjq7vg.7AhkpT73ke6dCcN9XNwSMw.E_L4WPiryeVCW_0jWiWMA6QjWu5gIIA4Ipysar9kOANIogUjr8Gwfxrj-0rW9h7gfeSbFaeQdNWKPAwlnkYIzB3Na6nUALnyFY-ki1euFGU.F3Zs36mJlugS4fB-6CtqbQ"
+        },
+    
+        "record": {
+            "CNAME": "asteral1.github.io"
```

Comment:
404 GitHub page. Please have content or send a preview.

Reply:
![Screenshot_2024-10-06-01-34-14-365_com android chrome](https://github.com/user-attachments/assets/65a41d6e-3022-496e-9389-70a9b15d3777)

Reply:
Done!

Reply:
no, not done, it's still a 404

Reply:
I updated the asteral.json

Reply:
The site can be accessed at https://asteral.github.io/asteral.github.io